### ISSUE TITLE
Possible closure_end_indentation rule issue

### ISSUE THREAD

**Issue by erichoracek** [realm/SwiftLint#2121]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Rule issue
I'm wondering whether the following code should be triggering the `closure_end_indentation` rule to fail (it currently doesn't):

```swift
function(
  parameter: a,
  closure: { x in
    // code
},
  anotherClosure: {
    // code
})
```

To me, this reads as a violation of the rule: since the closure's opening brace is on a line with one level of indentation, it seems like the closing brace should have the same level of indentation. The rule reads as follows:

> "Closure end should have the same indentation as the line that started it.

My assumption from reading the rule is that the following code would satisfy it:

```swift
function(
  parameter: a,
  closure: { x in
    // code
  },
  anotherClosure: {
    // code
  })
```

From this, it would appear that currently, the rule treats "the line that started it" as the start of the function declaration, rather than the line on which the opening brace is located.

What do you think? Thanks in advance! cc @marcelofabri

**Comment by jpsim** [comment]
I agree with this, I'd consider the existing behavior to be a bug and the correct way to address is to fix the existing behavior, no need to create a new rule or configuration point.
